tests/Unit/HC/amdgcn_ds_bpermute.cpp

```diff
@@ -77,7 +78,7 @@ int main() {
 
   pass &= run_test<unsigned int>(1024*1024);
   pass &= run_test<int>(1024*1024);
-  pass &= run_test<float>(1024*1024);
+  //pass &= run_test<float>(1024*1024);
```

Comment:
was turned off to debug the uniform_int_distribution issue with floats.  It works now with the change above.  I'll submit a commit to re-enable it.